resources/sts/4.11/sts_installer_core_permission_policy.json

```diff
@@ -0,0 +1,174 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+		    "autoscaling:DescribeAutoScalingGroups",
+		    "ec2:AllocateAddress",
+		    "ec2:AssociateAddress",
+		    "ec2:AttachNetworkInterface",
+		    "ec2:AuthorizeSecurityGroupEgress",
+		    "ec2:AuthorizeSecurityGroupIngress",
+		    "ec2:CopyImage",
+		    "ec2:CreateNetworkInterface",
```

Comment:
I believe this and the other network interface actions are used for the NAT gateways which would not be required for BYO VPC. Can you move and test that?

These these 

```
"ec2:AttachNetworkInterface",
"ec2:CreateNetworkInterface",
"ec2:DeleteNetworkInterface",
"ec2:DescribeNetworkInterfaces",
"ec2:ModifyNetworkInterfaceAttribute",
```

Reply:
I removed the `"ec2:CreateNetworkInterface"` permission and tried to create a BYO VPC cluster and got this error in the install logs:
`error creating EC2 Network Interface: UnauthorizedOperation: You are not authorized to perform this operation`

Reply:
More info about the failure and the cluster spec:
https://issues.redhat.com/browse/SDAINT-2993

Reply:
Ok lets leave these then.